app/addons/documents/header/header.actions.js

```diff
@@ -42,9 +42,10 @@ function (app, FauxtonAPI, ActionTypes) {
       });
     },
 
-    enableTableView: function () {
+    enableTableView: function (displayType) {
       FauxtonAPI.dispatch({
-        type: ActionTypes.ENABLE_TABLE_VIEW
+        type: ActionTypes.ENABLE_TABLE_VIEW,
+        displayType: displayType
```

Comment:
Not a big deal, but how do you feel about always using an `options` key here? i.e. putting all payload content into an `options` sub-object. I like it because it differentiates "special" keys (really only `type` right now) from arbitrary message content. Seems like a good gamble for down the road.